src/applications/lgy/coe/containers/EligibilityApp.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import { isLoggedIn } from 'platform/user/selectors';
+
+import { generateCoe } from '../actions';
+import { CertificateDownload } from '../components/CertificateDownload';
+import { CALLSTATUS, COE_FORM_NUMBER } from '../constants';
+
+const EligibilityApp = props => {
+  const {
+    router,
+    loggedIn,
+    certificateOfEligibility: { generateAutoCoeStatus, profileIsUpdating, coe },
+    hasSavedForm,
+  } = props;
+
+  const clickHandler = () => {
+    props.generateCoe('skip');
+  };
+
+  useEffect(
+    () => {
+      if (!loggedIn) {
+        router.push('/');
+      } else if (!profileIsUpdating && loggedIn && !hasSavedForm && !coe) {
+        props.generateCoe();
+      }
+    },
+    [loggedIn],
+  );
+
+  let content;
+
+  if (generateAutoCoeStatus === CALLSTATUS.idle || profileIsUpdating) {
+    content = <LoadingIndicator message="Loading application..." />;
+  } else if (generateAutoCoeStatus === CALLSTATUS.pending) {
+    content = (
+      <LoadingIndicator message="Checking automatic COE eligibility..." />
+    );
+  } else if (
+    generateAutoCoeStatus === CALLSTATUS.success ||
+    (generateAutoCoeStatus === CALLSTATUS.skip && coe)
```

Comment:
If a user has gone through the COE check, seen their COE but then opted to fill out the form, this allows them access back to their COE using the browser back button.